.vscode/settings.json

```
@@ -12,6 +12,7 @@
     "packages/_private/stylelint-config-syn",
     "packages/assets",
     "packages/components",
+    "packages/css",
```

should be `packages/styles`

```suggestion
    "packages/styles",
```